include/dt-bindings/pinctrl/microchip_pic32cxsg_pinctrl.h

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
This folder is at the top level.  Most hal_vendor folders have subdirectories containing the whole chip family. mec, mec5, mpfs contain all the specific stuff for those chip families.  Is it possible for you to put your stuff in a subdirectory also, so we are all consistent and isolated?